translationstats.py

```diff
@@ -13,10 +13,7 @@ class MacroType(Enum):
     OBJECT_LIKE = auto()
     FUNCTION_LIKE = auto()
 
-class TranslatorAction(Enum):
-    pass
-
-class SkipType(TranslatorAction):
+class TechnicalSkip(Enum):
     NOT_INTERFACE_EQUIVALENT = auto()
```

Comment:
I think you can remove this skip record type since it seems unused.

Reply:
It's still used in `MacroTranslator::should_skip_due_to_technical_limitations`

Reply:
Oh my bad - you mean `NOT_INTERFACE_EQUIVALENT`, good catch